### DISCUSSION TITLE
Сервисы (их окончание)

### DISCUSSION THREAD

**Discussion by ttm-alexeypolunin** [totumonline/totum-ru-issues-and-discussions#537] [Анонсы]
🪦 RIP

Закрыли внешнюю регистрацию на сервисы — они оказались никому не нужны.

Если вдруг вам нужны — ожидайте когда откроем локальную установку. Для этого надо сделать кучу всего (репо, установщик, новую схему тотума для управления итп).

Возможно во второй половине года, но это не точно.

Всем добра!

**Comment by lutikoff** [comment]
Печально, импорта и и экспорта xlsx не хватает, встал на этом

**Reply by ttm-alexeypolunin** [reply]
У market fit две части: возможности и оплата. С оплатой что-то не вышло. Зато вопросов не было. Те пока Сервисы отсутствовали — этот вопрос с экселем приходил раз в 3 дня. Запустили Сервисы — как отрезало, но платящих пользователей за два года их работы там было 2 человека.

**Reply by lutikoff** [reply]
Я уже 3й раз подхожу к тотуму и в основном из-за экспорта и импорта, кодом пытаюсь осилить ), если появится локально будет прекрасно.

**Reply by ttm-alexeypolunin** [reply]
Да он будет локально, но 1. установщик будет рассчитан под отдельную виртуалку 2. плата от микроволновки в качестве сервера для них не подходит, тк этим генераторам нужны и процессор и оперативка.

**Reply by lutikoff** [reply]
Ну с этим точно проблем нет, ресурсы позволяют.
